python/plugins/processing/algs/qgis/ExtentFromLayer.py

```diff
@@ -51,6 +53,7 @@ class ExtentFromLayer(QgisAlgorithm):
 
     INPUT = 'INPUT'
     BY_FEATURE = 'BY_FEATURE'
```

Comment:
Just because I'm on it now, could I remove this line? I have no clue what it could do...